docs/gen-ai/gen-ai-spans.md

```diff
@@ -322,6 +328,20 @@ Instrumentations SHOULD document the list of errors they report.
 |---|---|---|
 | `_OTHER` | A fallback error value to be used when the instrumentation doesn't define a custom value. | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
 
+---
+
+`gen_ai.operation.name` has the following list of well-known values. If one of them applies, then the respective value MUST be used; otherwise, a custom value MAY be used.
```

Comment:
this markdown is auto-generated, please update this yaml definition https://github.com/open-telemetry/semantic-conventions/blob/06536459c81f8d1d83d53bd04ab6638cb1315c42/model/gen-ai/spans.yaml#L264

and follow https://github.com/open-telemetry/semantic-conventions/blob/main/model/README.md to regenerate markdown